sale_amount_payment_link/tests/test_sale_amount_payment_link.py

```diff
@@ -1,11 +1,13 @@
 # Copyright 2021 Camptocamp SA
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
+from odoo.tests import tagged
 from odoo.tests.common import Form
 
 from odoo.addons.account.tests.common import AccountTestInvoicingCommon
 
 
+@tagged("post_install")
```

Comment:
@ilyasProgrammer Could you remove unrelated module modifications ?

Reply:
Ok, I see, it's to fix tests. But we can do it in another PR in order to merge it fast. Could you do it ?

Reply:
ok

Reply:
https://github.com/OCA/sale-workflow/pull/1989

Reply:
@ilyasProgrammer rebase?